doc/reference/guidelines.rst

```diff
@@ -0,0 +1,48 @@
+.. highlight:: python
+
+=================
+Coding Guidelines
+=================
+
+Python
+======
+
+- PEP8 ignore E501
+- prefer % over format
+- try to avoid generators and decorators
+
+
+File naming
+===========
+
+Split the business logic by sets of models, in each sets select a main
+model, this model gives its name to the set. If there is only one set of module
+it's name is the same as the module name.
```

Comment:
I am not sure to understand what you meant. How I understand it (and in that case I agree)

my_module/models/sale.py -> holds inheritance of sale order, sale order line
my_module/models/module.py -> holds my custom models